README.md

```diff
@@ -206,6 +206,10 @@ The CLI caches details of your projects and their environments, and some other
 information. These caches could become out-of-date. You can clear caches with
 the command `platform clear-cache` (or `platform cc` for short).
 
+Default cache lifetimes have now been reduced thanks to performance
```

Comment:
"have now been reduced" reads more as part of a changelog than in the readme, no?

Reply:
I could just remove this section..